pkg/Makefile

```diff
@@ -32,7 +34,8 @@ deb: prep
 		--license $(LICENSE) \
 		--url $(URL) \
 		--vendor $(VENDOR) \
-		--package $(TARGET_DIR)/aerospike-prometheus-exporter-$(VERSION)-amd64.deb
+		--architecture $(DEB_PKG_ARCH) \
+		--package $(TARGET_DIR)/aerospike-prometheus-exporter-$(VERSION)-$(DEB_PKG_ARCH).deb
```

Comment:
Did you also test these `fpm` builds - `rpm`, `deb`, `tar`?

Reply:
Yes, tested 97e07ac on arm64/amd64 deb and rpm systems, all installs worked fine.

Reply:
Great.